.golangci.yml

```
@@ -81,3 +81,9 @@ issues:
       text: "^G404:"
       linters:
         - gosec
+    - path: "(^|/)mocks($|/)"
+      linters:
+        - all
+    - path: .pb
```

I just meant the generated go files:
```suggestion
    - path: .pb.go
```
Were the `.pb` source files being flagged too?

---

I haven't seen either of them flagged by anything.  I just added them since you requested they be added to the ignore list... I assumed that was because they're generated files.  (Leaving off the .go was just a typo on my part.)

Did something flag them as problematic?

---

The generated `.pb.go` files were being reformatted at least. I was just trying to understand why the `.pb` source files were being included here, since I would be surprised if they get scanned at all as non-go files.